pkg/services/folder/folderimpl/folder_unifiedstorage.go

```diff
@@ -721,11 +721,6 @@ func (s *Service) moveOnApiServer(ctx context.Context, cmd *folder.MoveFolderCom
 		return nil, folder.ErrBadRequest.Errorf("missing signed in user")
 	}
 
-	// k6-specific check to prevent folder move for a k6-app folder and its children
-	if cmd.UID == accesscontrol.K6FolderUID {
```

Comment:
Don't we still need this check ?

Reply:
Good catch, thank you! It was something later down we could remove due to the code you linked:
```
	if f != nil && f.ParentUID == accesscontrol.K6FolderUID {
		return nil, folder.ErrBadRequest.Errorf("k6 project may not be moved")
	}
```

Updated!